tests/codeception/e2e/extensions/UserCreateCest.php

```diff
@@ -28,9 +28,7 @@ public function tryToTest(E2eTester $I)
 
         $I->click('button[type="submit"]');
 
-        $I->waitForText('Success');
+        $I->waitForText('Success',10, '#growls-br > .growl.growl-notice');
```

Comment:
Growl is sometimes a little bitchy about timing. This ensures that the test does not fail due to a slow pipeline.